Document aliases (#528)
cirruslabs/cirrus-ci-docs: docs/guide/writing-tasks.md

  script: yarn run publish
```

??? tip "Task Names and Aliases"

>     It is possible to specify task name via `name` field. `lint_task` syntax is simply a syntactic sugar that will be

Reviewer: ```suggestion
    It is possible to specify the task's name via the `name` field. `TASKNAME_task` syntax is simply a syntactic sugar that will be
```

Author: the `lint_task` pairs with the code example below to show how it expands.

Reviewer: Got it.

    Names can be also pretty complex:

    ```yaml
    task:
      name: Tests Shard $TESTS_SPLIT
      env:
        matrix:
          TESTS_SPLIT: 1/3
          TESTS_SPLIT: 2/2
          TESTS_SPLIT: 3/3
      tests_script: ./.ci/tests.sh
    
    deploy_task:
      only_if: $CIRRUS_BRANCH == 'master'
      depends_on:
        - Tests Shard 1/3
        - Tests Shard 2/3
        - Tests Shard 3/3
      script: ./.ci/deploy.sh
      ...
    ```
    

>     Complex task names make it harder to list and **maintain** all of such task names in `depends_on` field. In order to 

Reviewer: ```suggestion
    Complex task names make it difficult to list and **maintain** all of such task names in your `depends_on` field. In order to 
```

>     make it simpler you can use `alias` field to have a short simplified name for several tasks to use in `depends_on`.

Reviewer: ```suggestion
    make it simpler you can use the `alias` field to have a short simplified name for several tasks to use in `depends_on`.
```

>     Here is an example of a modified example above that leverages `alias` field:

Reviewer: ```suggestion
    Here is a modified version of an example above that leverages the `alias` field:
```

    
    ```yaml hl_lines="3 13"
    task:

>       name: Tests Shard $TESTS_SPLIT

Reviewer 2: ```suggestion
      name: Test Shard $TESTS_SPLIT
```
Test is an adjective

Reviewer 2: Sorry, I was late to the party

Reviewer: hang on a second
I missed something?!

😠 impossible

Author: Nope, seems names are consistent, for example: `Tests Shard 1/3`

Reviewer 2: > Nope, seems names are consistent, for example: `Tests Shard 1/3`

It could be consistent, but it is not grammatically correct, That's what I mean. 
in "Test shard", shard is the noun, and test is the adjective, therefore it cannot be tests since adjetives cannot be expressed in plural.

At least that's how I interpret it. But if you mean "Tests - Shard 1/3" I would say that one is correct

Reviewer: Yeah he's right, that's not grammatically correct
